make/modules/java.base/Lib.gmk

```diff
@@ -80,6 +80,26 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBNIO, \
     NAME := nio, \
     OPTIMIZATION := HIGH, \
     WARNINGS_AS_ERRORS_xlc := false, \
+    Net.c_CFLAGS := \
+        -I$(OPENJ9OMR_TOPDIR)/include_core \
+        -I$(OUTPUTDIR)/vm/runtime/jcl, \
+    nio_util.c_CFLAGS := \
+        -I$(OPENJ9_TOPDIR)/runtime/include \
+        -I$(OPENJ9_TOPDIR)/runtime/oti \
+        -I$(OPENJ9OMR_TOPDIR)/include_core \
+        -I$(OUTPUTDIR)/vm/runtime \
```

Comment:
Uses of `$(OUTPUTDIR)/vm/runtime` should be replaced by `$(OPENJ9_VM_BUILD_DIR)`, in part to enable UMA builds, but the include paths are incomplete for UMA.